pathfinder/tests/test_blockchain_integration.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+
+"""
+The test in this module uses the mocked raiden client to create blockchain events and
+processes them. Additionally, it mocks the transport layer directly. It tests the
+interaction of many moving parts - yet, it is currently really slow.
+Therefore, usually mocked_integration should be used.
+"""
+
+import gevent
+from raiden_contracts.contract_manager import ContractManager
+from pathfinder.pathfinding_service import PathfindingService
+from pathfinder.tests.fixtures import Mock
+from raiden_libs.blockchain import BlockchainListener
+
+
+def test_pfs_with_mocked_client(
+    generate_raiden_clients,
+    ethereum_tester,
+    contracts_manager: ContractManager,
+    blockchain_listener: BlockchainListener
+
+):
+    """ instantiates a pathfinding service with mocked transport,
+    listening and porcessing blockchain events created by mocked Raiden clients"""
+    clients = generate_raiden_clients(7)
+    network_address = clients[0].contract.address
+
+    pathfinding_service = PathfindingService(
```

Comment:
Why don't you use the PFS fixture that was added before? If not needed please remove the fixture.

Reply:
Done.